webapp/client/src/pages/TelephoneNumberPage.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import { fieldPropType } from "../lib/propTypes";
+import FormFieldTextInput from "../components/FormFieldTextInput";
+
+export default function TelephoneNumberPage({
+  stepHeader,
+  form,
+  fields,
+  prevUrl,
+}) {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <StepHeaderButtons url={prevUrl} />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form}>
+        <FormFieldTextInput
+          autofocus
```

Comment:
Do we really want to set autofocus here? Is that still part of the design spec?

Reply:
Good question.

Reply:
Nadine said no^^